yaksh/models.py

```diff
@@ -134,6 +140,346 @@ def create_trial_course(self, user):
     def get_hidden_courses(self, code):
         return self.filter(code=code, hidden=True)
 
+
+#############################################################################
+class Lesson(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    html_data = models.TextField(null=True, blank=True)
```

Comment:
So where is the markdown saved?  Or is that in the description?  Perhaps a comment before the field may be useful for a reader?

Reply:
markdown is saved in the description and html_data contains converted HTML text. Will add comments.